docs/API/index.md

```diff
@@ -10,7 +10,6 @@
 * [folder](folder.md)
 * [librarypanel](librarypanel.md)
 * [panel](panel/index.md)
-* [playlist](playlist/index.md)
```

Comment:
I just saw this when going through the changes, is it expected that Playlists no longer exist?

Reply:
Yes, that got removed from the schemas.